tool/travis.sh

```diff
@@ -5,19 +5,52 @@
 # found in the LICENSE file.
 
 # Fast fail the script on failures.
-set -e
+set -ex
```

Comment:
What's the difference between this and `-e`?

Reply:
`-e` quits immediately upon error (actually sometimes too quick, I've seen it quit before the output was flushed in some of the CI builds).

`-x` prints the commands as it runs them (with some variable expansion). I added this to track down some issues while debugging this so it isn't critical - I can remove it if we think it's noisy. I'll land with it in since I think it's useful if the builds are flaky (which it's possible they could be with us running it more on a new OS) but I'm happy to remove it if it seems unnecessary or noisy.